### ISSUE TITLE
Weird character in the title

### ISSUE THREAD

**Issue by mo0rti** [onfido/onfido-android-sdk#223]
#### What was the expected behaviour?
An empty toolbar title should be shown.

#### What happened instead?
A wired char appears on the toolbar. See the attached image.

#### Version info:

- `onfido-capture-sdk` version: 13.0.1
- Android API version(s): 31

- `compileSdkVersion`: 33
- `targetSdkVersion`: 33
- `minSdkVersion`: 29
- Support library version (if applicable):

#### Steps to reproduce:
I tried to start Onfido and there is a weird character is displayed on the toolbar.

#### Onfido Flow initialization code:
```
    private fun createCaptureScreenStep(): FlowStep {
        val country = CountryCode.NL
        val builder: BaseDocumentCaptureScreenStepBuilder = DocumentCaptureStepBuilder.forNationalIdentity()
        if (builder is DocumentCaptureWithCountryCodeScreenStepBuilder<*>) {
            builder.withCountry(country)
        }
        return builder.build()
    }

    fun startFlow(requestCode: Int, token: String, certificatePubKeyHash: String) {
                val onfido = activity.let { OnfidoFactory.create(it).client }

        val onfidoDocumentTypes = listOf(
            DocumentType.PASSPORT,
            DocumentType.NATIONAL_IDENTITY_CARD,
            DocumentType.DRIVING_LICENCE
        )

        val onfidoConfig = activity.let {
            OnfidoConfig.builder(it)
                .withSDKToken(token)
                .withCustomFlow(listOf(createCaptureScreenStep()).toTypedArray())
                .withCertificatePinning(arrayOf(certificatePubKeyHash))
                .withAllowedDocumentTypes(onfidoDocumentTypes)
                .build()
        }

        onfido?.startActivityForResult(activity, requestCode, onfidoConfig)

    }

```

#### Other useful info:
![Screen Shot 2022-11-30 at 20 03 33](https://user-images.githubusercontent.com/866090/204886034-cef50ad7-53ef-4e3a-9a4e-67a1aa86dd3f.png)

**Comment by kushal-onfido** [comment]
Hello @mo0rti 

This case seems similar to https://github.com/onfido/onfido-android-sdk/issues/209 and our team is working on it. We'll update you here once this is fixed. 

Thank you!

**Comment by kushal-onfido** [comment]
Hello @mo0rti 

This issue has been fixed in the SDK release 15.0.0. Please refer to the release notes [here](https://github.com/onfido/onfido-android-sdk/releases/tag/15.0.0).

Thank you!

**Comment by sokarcreative** [comment]
Exact same character on version 17.0.0. This character is shown for a second then no title is displayed
